### PR TITLE
Adjust Security Loadouts, adds alternative firearms

### PR DESCRIPTION
**Pull request by Lyndomen**
## About the PR
Removed MK58 from security equipment, and removed spare magazine.  Removes Inspector & speedloaders from detective belt. 

Adds new loadout category for HOS, Warden, Corpsman, Prison Guard, & Officer: **Security Firearm**
- Pollock 
- SLP-57
- Mk 58
- disabler

Adds new loadout category for HOS, Warden, Corpsman, Prison Guard, & Officer: **Ammunition**
- .35 lethal
- .35 rubber
- .38 special lethal speedloader
- .38 special rubber speedloader

Adds new loadout category for Detective: **Security Revolver**
- Inspector
- fitz special
- k-38 masterpiece
- Lucky Revolver

Adds new loadout category for Detective: **Ammunition**
- .45 magnum speedloader
- .45 magnum rubber speedloader

Cadets also gain a disabler. 

## Why / Balance
More gun options is cool. None of these dramatically adjust balance, if anything they are worse than the MK58. Compare guns & talk to people. Bring your own gun to work day

## Technical details
YAML loadouts

## Media
image
image
image
image
image
image

## Requirements
- [X] I have tested all added content and changes.
- [X] I have added media to this PR or it does not require an ingame showcase.

## Breaking changes
no

**Changelog**
:cl:
- add: Security roles may select their service weapon in loadouts, along with their ammunition of choice

### CODE REVIEW

File: Resources/Locale/en-US/deltav/preferences/loadout-groups.ftl [hunk 1]
```diff
@@ -93,6 +93,12 @@ loadout-group-brig-medic-outerclothing = Corpsman outer clothing
 loadout-group-prison-guard-head = Prison Guard head
 loadout-group-prison-guard-jumpsuit = Prison Guard jumpsuit
 
+loadout-group-security-gun = Security Firearm
+loadout-group-revolver-gun = Security Revolver
+loadout-group-security-gun-ammo = Ammunition
+loadout-group-revolver-ammo = Ammunition
+
+
```
Comment (deltanedas) [inline Resources/Locale/en-US/deltav/preferences/loadout-groups.ftl:102]: extra line bad

File: Resources/Prototypes/DeltaV/Roles/Jobs/Security/brigmedic.yml [hunk 2]
```diff
@@ -34,7 +34,7 @@
     id: CorpsmanPDA
     ears: ClothingHeadsetBrigmedic
     belt: ClothingBeltCorpsmanWebbingFilled
-    pocket1: WeaponPistolMk58Nonlethal
+ #  pocket1: WeaponPistolMk58Nonlethal # DeltaV loadouts
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Roles/Jobs/Security/brigmedic.yml]: `DeltaV - loadouts` for these

File: Resources/Prototypes/Loadouts/Jobs/Security/detective.yml [hunk 3]
```diff
@@ -76,3 +76,27 @@
   id: StasecWinterCoatDet
   equipment:
     outerClothing: ClothingOuterCoatStasecDet
+
+- type: loadout # DeltaV
```
Comment (deltanedas) [inline Resources/Prototypes/Loadouts/Jobs/Security/detective.yml]: put these in the deltav folder

File: Resources/Prototypes/Loadouts/Jobs/Security/security_officer.yml [hunk 4]
```diff
@@ -204,3 +204,46 @@
   storage:
     back:
     - Dinkystar
+
+# Guns DeltaV
+- type: loadout # DeltaV
```
Comment (deltanedas) [inline Resources/Prototypes/Loadouts/Jobs/Security/security_officer.yml]: .

File: Resources/Prototypes/Loadouts/role_loadouts.yml [hunk 5]
```diff
@@ -383,8 +385,10 @@
   - SecurityShoes
   - SurvivalSecurity
   - Trinkets
-  - SecurityStar 
+  - SecurityStar
```
Comment (deltanedas) [inline Resources/Prototypes/Loadouts/role_loadouts.yml]: sus

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by ThataKat** [comment]
Pollock my beloved. Maybe some of these should have playtime requirements so only cool kids get cool guns?

**Comment by Radezolid** [comment]
The .38 guns should be Senior Officer minimum IMO

File: Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml [hunk 6]
```diff
@@ -157,7 +200,7 @@
   - ChiefJusticeJumpskirt
   - ChiefJusticeFormalJumpsuit
   - ChiefJusticeWhiteJumpsuit
-  
+
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml]: untroll

File: Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml [hunk 7]
```diff
@@ -186,7 +229,7 @@
   loadouts:
   - ClerkJumpsuit
   - ClerkJumpskirt
-  
+
```
Comment (deltanedas) [inline Resources/Prototypes/DeltaV/Loadouts/loadout_groups.yml]: untroll

File: Resources/Prototypes/Loadouts/role_loadouts.yml (hunk 5 above)
Comment (deltanedas) [reply Resources/Prototypes/Loadouts/role_loadouts.yml]: this line is still changed erroneously

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Comment by deltanedas** [comment]
gatekeeping guns is already done with cadet vs secoff :trollface:

**Comment by Lyndomen** [comment]
I don't love gatekeeping things with mechanical differences behind hours played, can feel a bit shit imo for new players

**Review by deltanedas — DISMISSED** [review]
approved codewise but the guns balance need to be looked at so theres not a clear meta

**Comment by Lyndomen** [comment]
> approved codewise but the guns balance need to be looked at so theres not a clear meta

Real was gonna nerf .45 in another PR soon, gotta test it more. Or just axe universals idk, op ass guns

**Comment by CybersunOperative** [comment]
Hi, Hello. Direction here (val). We would rather not give regular officer's .38s. We are more receptive to giving Detective access to .38 revolvers (And I personally recommend the "Lucky 37" since it looks cooler and is pretty much the same as the K-38.)

I request that the Snubnose is removed from an option. It's performance is questionable and unbefitting security. 

I am unsure if you are aware, but the K-38 loads with rubbers already loaded. This is fine, but you might want to change that so it's in line with the other weapons.

HoS/Warden should also have access to the revolver options that the detective has.

after that, it should be alright. more loadout options for security is based.

**Comment by Lyndomen** [comment]
HOS Warden Loadout: 
image


Det loadout: 
image


Officer loadout: 
image


38 special guns now spawn with lethals: 
image

**Comment by CybersunOperative** [comment]
I have no further balance questions, seems fine to merge if Deltanades says the code is fine. No one else in direction has commented on this PR who I haven't already voiced the opinion of.

File: Resources/Prototypes/Roles/Jobs/Security/security_cadet.yml [hunk 8]
```diff
@@ -34,6 +34,7 @@
     ears: ClothingHeadsetSecurity
     belt: ClothingBeltSecurityFilled
 #    pocket1: WeaponPistolMk58Nonlethal # DeltaV - Security Cadet doesn't spawn with a gun
+    pocket1: WeaponDisabler # DeltaV - loadouts
```
Comment (deltanedas) [inline Resources/Prototypes/Roles/Jobs/Security/security_cadet.yml]: comment should stay the same

**Review by deltanedas — CHANGES_REQUESTED** [review]

**Review by deltanedas — APPROVED** [review]

**Pull request merged**